src/ogt_voxel_meshify.h

```diff
@@ -134,9 +134,14 @@ typedef void  (*ogt_voxel_meshify_free_func)(void* ptr, void* user_data);
 // a context that allows you to override various internal operations of the below api functions.
 struct ogt_voxel_meshify_context
 {
-    ogt_voxel_meshify_alloc_func                alloc_func;                 // override allocation function
-    ogt_voxel_meshify_free_func                 free_func;                  // override free function
-    void*                                       alloc_free_user_data;       // alloc/free user-data (passed to alloc_func / free_func )
+    ogt_voxel_meshify_alloc_func alloc_func;                 // override allocation function
+    ogt_voxel_meshify_free_func  free_func;                  // override free function
+    void*                        alloc_free_user_data;       // alloc/free user-data (passed to alloc_func / free_func )
+
+    // for each tessellated voxel that produces 1 or more vertices, call an optionally
+    // provided user function which allows them to modify the vertex information if needed 
+    void                         (*emit_verts_cb)(uint32_t x, uint32_t y, uint32_t z, ogt_mesh_vertex* verts, uint32_t count, void* cb_ctx);
```

Comment:
I would recommend typedeffing the function pointer type in the same way ogt_voxel_meshify_alloc_func/free_func are done above.